src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -423,10 +423,14 @@ protected IEnumerable<WorkloadId> WriteSDKInstallRecordsForVSWorkloads(IEnumerab
 
     internal static class InstallingWorkloadCommandParser
     {
-        public static readonly CliOption<string> WorkloadSetVersionOption = new("--version")
+        public static readonly CliOption<string> WorkloadSetVersionOption = new CliOption<string>("--version")
         {
             Description = Strings.WorkloadSetVersionOptionDescription
-        };
+        }.AddCompletions((ctx) =>
+        {
+            var versions = Search.Versions.SearchWorkloadSetsCommand.GetWorkloadSetVersions(ctx).GetAwaiter().GetResult();
+            return versions.Select(v => new System.CommandLine.Completions.CompletionItem(v));
+        });
```

Comment:
Here's the completions hookup - we needed something that could take the completion-time parse result and turn it into CompletionItems.